draft.md

```diff
@@ -129,45 +129,45 @@ The native Emotiq token (EMTQ) has deflationary economics, with the token supply
 
 ## 4.6 Implementation Details
 
-Emotiq will release a technical paper (yellowpaper) with full implementation details in the next 1-2 months.
+Emotiq has released a technical paper (yellow paper) with full implementation details, available on the website.
 
 # 5. Token Generation Event
 
-Emotiq plans to raise $60M (million) through the combination of a private sale and a public token generation event (TGE). The fundraising target has been carefully selected to enable us to reach our development milestones and jumpstart the Emotiq ecosystem. Emotiq will generate one billion EMTQ tokens, distributed as follows:
+Emotiq plans to raise $39M (million) through multiple private sale rounds. The fundraising target has been carefully selected to enable us to reach our development milestones and jumpstart the Emotiq ecosystem. Emotiq will generate one billion EMTQ tokens, distributed as follows:
 * 15% sold to seed contributors (completed Dec 2017);
-* 14% sold during the private sale;
-* 36% sold during the public TGE; and
-* 35% to be held by the Emotiq team, to be used as detailed later in this section.
+* 34% sold during the private sales;
```

Comment:
Can you double-check the numbers? We are selling 51% of the tokens to the public and the numbers do not add up to 49%.